.gitignore

```diff
@@ -1,5 +1,6 @@
 # Build/deps dir
 /build/
+/cmake-build-debug/
```

Comment:
where'd this come from?

Reply:
I'm not sure.. i'll just delete the folder for now i guess

Reply:
CLion creates build folders with this name, I think.

Reply:
oh, we can ignore if it's coming from a build tool and not a personal config.